### ISSUE TITLE
[BUG] Missing ability to only search in stop places

### ISSUE THREAD

**Issue by mikaelbr** [AtB-AS/mittatb-app#1046]
## Origin

_Links to received feedback, user research or other findings._

> Jeg tror også det hadde vært bedre om det først kom opp buss-stopp når en bruker søke-funksjonen. For eksempel der det nå står «Siste steder» og «Søkeresultater» kunne det også stått en avdeling mellom de to kalt «Holdeplasser». Slik blir det enda tydeligere når en søker at en velger et buss-stopp, og ikke et sted på kartet.
> 
> (Vet at det er et buss-ikon, men det har skjedd flere ganger at jeg trodde jeg valgte busstopp, men så var det et sted på kartet, og da får jeg opp sånn «gå-mann» i veibeskrivelsen min, og det vil jeg ikke ha) :)

From https://app.intercom.com/a/apps/vdemedo2/inbox/inbox/all/conversations/159245400006386

## Motivation

_A short description of what user needs or business goals this feature will solve._

## Hypotheses and assumptions

_A list of hypotheses and assumptions we have made about the user or the proposed solution._

## Proposed solution

_A coarse description of a proposed solution, that may include wireframes and graphic design._

## Acceptance Criteria

_List of relevant acceptance criteria as a part of a QA flow_

- [ ] Renders colors as expected in dark and light mode.
- [ ] Is translatable to English and Norwegian.
- [ ] Does not drain battery
- [ ] Works in up to 200% font size
- [ ] Is operable and perceivable using screen reader

**Comment by pettersuul** [comment]
I believe the proposed solution by the user makes the search more complicated and will use to much space and move other users relevant results . However, maybe it could be a setting the user can enable.

**Comment by mikaelbr** [comment]
I agree. I don't see how this best can be handled without for instance making the search modal much more complex/advanced with tabs, settings etc.

**Comment by monopoint** [comment]
Kanskje vi kan kommunisere tydeligere forskjellen mellom holdeplass og vanlig sted? Jeg støtter brukeren litt her, og synes ikke at buss-ikonet overføres så godt til holdeplass. jeg er allerede i buss-modus, og tenker ikke over at et treff vises med buss-ikon, og skjønner dermed ikke at dette ikonet skal gi meg informasjon om hva slags type sted det er.
Forslag: 
- Juster ikonet til å ligne mer på noe man forbinder med en holdeplass
- Ha en toggle til høyre for overskriften: "vis bare holdeplasser"
